helpdesk_timesheet/views/helpdesk_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="helpdesk_ticket_view_inherit_timesheets" model="ir.ui.view">
+        <field name="name">Helpdesk with timesheets</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='user_id']" position="after">
```

Comment:
Why using xpath here?

Reply:
i dont see any improvement by this but i can change it